rules/REQUEST-932-APPLICATION-ATTACK-RCE.conf

```diff
@@ -1113,6 +1113,53 @@ SecRule REQUEST_HEADERS:Referer "@rx ^[^\.]*?(?:['\*\?\x5c`][^\n/]+/|/[^/]+?['\*
             setvar:'tx.rce_score=+%{tx.critical_anomaly_score}',\
             setvar:'tx.inbound_anomaly_score_pl2=+%{tx.critical_anomaly_score}'"
 
+
+#
+# -=[ Rule 932207 ]=-
+#
+# Sibling of 932205 targeting fragments in the Referer header.
+#
+# The last chain prevents FPs against the "Scroll to text fragment" browser feature
+# (https://wicg.github.io/scroll-to-text-fragment/).
+#
+# Regular expression generated from regex-assembly/932207.ra.
+# To update the regular expression run the following shell script
+# (consult https://coreruleset.org/docs/development/regex_assembly/ for details):
+#   crs-toolchain regex update 932207
+#
+SecRule REQUEST_HEADERS:Referer "@rx ^[^#]+\#[^#]+" \
+    "id:932207,\
+    phase:1,\
+    block,\
+    capture,\
+    t:none,t:lowercase,t:urlDecodeUni,\
+    msg:'RCE Bypass Technique',\
+    logdata:'Matched Data: %{TX.2} found within %{TX.932207_MATCHED_VAR_NAME}: %{MATCHED_VAR}',\
+    tag:'application-multi',\
+    tag:'language-multi',\
+    tag:'platform-multi',\
+    tag:'attack-rce',\
+    tag:'paranoia-level/2',\
+    tag:'OWASP_CRS',\
+    tag:'capec/1000/152/248/88',\
+    tag:'PCI/6.5.2',\
+    ver:'OWASP_CRS/4.12.0-dev',\
+    severity:'CRITICAL',\
+    setvar:'tx.932207_matched_var_name=%{matched_var_name}',\
+    chain"
+    SecRule TX:0 "@rx ^[^\.]+\.[^;\?]+[;\?](.*(['\*\?\x5c`][^\n/]+/|/[^/]+?['\*\?\x5c`]|\$[!#\$\(\*\-0-9\?-\[_a-\{]))" \
+        "capture,\
+        t:none,\
+        chain"
+        SecRule MATCHED_VAR "@rx /" \
+            "t:none,\
+            chain"
+            SecRule MATCHED_VAR "@rx \s" \
+                "t:none,\
+                setvar:'tx.rce_score=+%{tx.critical_anomaly_score}',\
+                setvar:'tx.inbound_anomaly_score_pl2=+%{tx.critical_anomaly_score}'"
```

Comment:
You must include this [chain rule](https://github.com/coreruleset/coreruleset/commit/cf0bb6ea2000fa132716b8ac537e1e21fcaddce8#diff-2bc4203bbb0b15818a3fdfc562633aec97130a2c5ffe12cf7fe2baf4b9b39ba7R1031) to prevent FPs against "scroll-to-fragment" (see my [comment](https://github.com/coreruleset/coreruleset/issues/3498#issuecomment-2644861019)).